.github/workflows/prune.yml

```diff
@@ -0,0 +1,47 @@
+name: Prune GHCR
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+permissions:
+  contents: write
+  packages: write
+  security-events: write
+
+jobs:
+  prune_images:
+    name: Prune old sparrow images
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+      packages: write
+      security-events: write
```

Comment:
These permissions aren't necessary since you've declared them as default permissions above.

Ref: https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs#overview